tools/patch_libultra_math.c

```diff
@@ -79,6 +79,7 @@ struct ar_header {
 #define FLAGS_MIPS3 0x20
 #define FLAGS_O32ABI 0x100000 
 int main(int argc, char **argv) {
+    argc = 0;
```

Comment:
No. Why?

Reply:
argc isn't actually used anywhere for whatever reason. I don't get why argc was even in here to begin with. imo this function would need to be rewritten to remove the need for argc in the first place, as this is more of a workaround than anything else.

Reply:
`argc` and `argv` are passed to `main` by standard and convention and contain the argument count and argument values respectively. I don't really understand this whole thing with changing a `return 0` to `return variable that was just set to 0` either.
If you're concerned that it's unused, it would be a good idea to actually check the value of `argc` and abort if it's not at least 2.

Reply:
Removed as a result.